src/app/public/modules/page/page.component.html

```diff
@@ -0,0 +1,3 @@
+<main class="sky-theme-default">
```

Comment:
Just curious - why force the `main` tag? Is the intent here to always have our consumers wrap their pages with the `sky-pages` component moving forward? Could this get our consumers into trouble if they add multiple? Or accidentally next them?

If that's the idea, then we need to make sure our docs are very explicit and consumers understand the impact to the DOM of using this new component.

Reply:
Yes, all pages will be wrapped with `sky-page` going forward.